src/itest/resources/features/company-links.feature

```diff
@@ -1,13 +1,45 @@
 Feature: Process company links information
 
-  Scenario: Consume the message and creating the company links
+  Scenario: Consume the message and creating the company links successfully
 
     Given Company links consumer api service is running
     When a message is published for companyNumber "00006401" to update links
     Then the Company Links Consumer should send a PATCH request to the Company Profile API
 
-  Scenario: Consume the message and update the company links
+  Scenario: Consume the message and update the company links successfully
 
     Given Company links consumer api service is running
-    When a message is published for companyNumber "00006400" to check for links
+    When a message is published for companyNumber "00006400" to check for links with status code "200"
     Then the Company Links Consumer should send a GET request to the Company Profile API
+
+  Scenario: Consume invalid message
+
+    Given Company links consumer api service is running
+    When a non-avro message is published and failed to process
+    Then the message should be moved to topic "stream-company-insolvency-company-links-consumer-invalid"
```

Comment:
can you please separate happy path and error-retry stories into separate feature files?